resources/admin/filter.js

```diff
@@ -1,43 +1,10 @@
+import { addCloseListener, addInputListeners, csrfToken, destroySelected, icon, notifyUser } from "./utils.js";
```

Comment:
Not sure if this bug is new, but if a filter contains a `Participation` predicate, a title is required.

If you want, I can have a look at the js part.

Reply:
This was fixed with: https://github.com/uzh/pool/pull/133